web/src/api/stationAPI.ts

```diff
@@ -1,7 +1,7 @@
 import axios from 'api/axios'
 import { FireSeason } from 'api/percentileAPI'
 
-export interface Station {
+export interface WeatherStation {
```

Comment:
Having Station and WeatherStation is a bit confusing. The one is for the GeoJSON station list, the other is for the Station object from the other endpoints. They're both weather stations.... We either need better names for them, or be consistent in how the API returns them I think.